### DISCUSSION TITLE
Aggregate discussion for similar aricle

### DISCUSSION THREAD

**Discussion by Nittarab** [dailydotdev/daily#983] [Feature Requests]
It would be cool to have "news" coverage feature and aggregate the discussion 

Example:
https://app.daily.dev/posts/27k45rttG
https://app.daily.dev/posts/QvHuFmwSh

**Comment by idoshamun** [comment]
Hi @Nittarab,

That's a fantastic idea because it's already on our roadmap! 🎉
Stay tuned!

**Comment by Nittarab** [comment]
Nice, I can't wait to se it, if you need any feedback, let me know
